modules/solid_mechanics/include/materials/ReturnMappingModel.h

```diff
@@ -8,61 +8,79 @@
 #define RETURNMAPPINGMODEL_H
 
 #include "ConstitutiveModel.h"
+#include "NewtonIterationSingleVariable.h"
 
 /**
- * One or more constitutive models coupled together.
+ * Base class for models that perform return mapping iterations to compute stress.  This
+ * is for a single model.  This class can be used directly for a single model, or combined
+ * with other models.
```

Comment:
How combined with other models?  Will the multiple models through the picard iterations in tensor mechanics work in a similar fashion to how they do now?